Cargo.toml

```diff
@@ -22,21 +22,22 @@ members = [
 
 
 [features]
-default = ["compiler", "value", "diagnostic", "path", "parser", "stdlib", "datadog", "core"]
+default = ["compiler", "value", "diagnostic", "path", "parser", "parsing", "stdlib", "datadog", "core"]
```

Comment:
Does the `parsing` feature enable any different features or behavior by itself? I'm just wondering if it even needs to be here, given that the `datadog` feature enables it indirectly.

Reply:
ah, right, good catch!